src/App.js

```diff
@@ -218,6 +229,11 @@ const returnStartedRoute = (Component, props, rest) => {
     return <Redirect to={{pathname: '/wallet/'}}/>;
   }
 
+  const firstWallet = wallet.getFirstWalletPrefix();
```

Comment:
Maybe add a comment here. If I understood correctly, this is used when you've just reset your wallet, is that correct?